src/devices_models/devices/common/add_to_expression.jl

```diff
@@ -212,7 +212,7 @@ function add_to_expression!(
     U <: TimeSeriesParameter,
     V <: PSY.StaticInjection,
     W <: AbstractDeviceFormulation,
-    X <: StandardPTDFModel,
+    X <: Union{PTDFPowerModel, StandardPTDFModel},
```

Comment:
Same here. I think that the `PTDFPowerModel` just needs the addition to the nodals expressions.